doc/api/http.md

```diff
@@ -1661,10 +1661,17 @@ A collection of all the standard HTTP response status codes, and the
 short description of each.  For example, `http.STATUS_CODES[404] === 'Not
 Found'`.
 
-## http.createServer([requestListener])
+## http.createServer([options][, requestListener])
 <!-- YAML
 added: v0.1.13
 -->
```

Comment:
Can you add the fact that the `options` argument is supported now to the changelog? i.e.

```yaml
<!-- YAML
added: v0.1.13
changes:
  - version: REPLACEME
    pr-url: https://github.com/nodejs/node/pull/15752
    description: The `options` argument is supported now.
-->
```

(This could also be done when landing, I guess, so I’m not removing the `ready` label.)